cppcache/integration-test/testThinClientSecurityAuthorizationMU.cpp

```diff
@@ -919,47 +840,27 @@ DUNIT_TASK_DEFINITION(READER_CLIENT, StepThree)
     }
     HANDLE_NOT_AUTHORIZED_EXCEPTION
 
-    // ASSERT(!rptr->containsKey(keys[2]),   "Key should not have been found in
-    // the region");
-
     try {
       LOG("Trying updateEntry");
-      // updateEntry(regionNamesAuth[0], keys[2], nvals[2], false, false);
       rptr->put(keys[2], nvals[2]);
       FAIL("Should have got NotAuthorizedException during updateEntry");
     }
     HANDLE_NOT_AUTHORIZED_EXCEPTION
 
-    // ASSERT(!rptr->containsKey(keys[2]),  "Key should not have been found in
-    // the
-    // region");
-
     try {
-      //auto regPtr0 = getHelper()->getRegion(regionNamesAuth[0]);
-     auto keyPtr = CacheableKey::create(keys[2]);
-     auto checkPtr =
-         std::dynamic_pointer_cast<CacheableString>(rptr->get(keyPtr));
-     if (checkPtr != nullptr) {
-       char buf[1024];
-       sprintf(buf, "In net search, get returned %s for key %s",
-               checkPtr->value().c_str(), keys[2]);
-       LOG(buf);
-     } else {
-       LOG("checkPtr is nullptr");
-     }
-    }
-    HANDLE_NO_NOT_AUTHORIZED_EXCEPTION
-
-    //auto regPtr0 = getHelper()->getRegion( regionNamesAuth[0] );
-    if (rptr != nullptr) {
```

Comment:
Ripped this, but I don't know if it should be made to actually do something, instead.